core/src/main/java/io/kestra/core/models/triggers/types/Schedule.java

```diff
@@ -140,15 +137,6 @@ public class Schedule extends AbstractTrigger implements PollingTriggerInterface
     @Builder.Default
     private String timezone = ZoneId.systemDefault().toString();
 
-    @Schema(
-        title = "Backfill option in order to fill missing past dates.",
-        description = "Kestra could optionally handle a backfill. The concept of a backfill is to replay missing schedules when a flow is created but we need to schedule it before its creation date.\n" +
-            "\n" +
-            "A backfill will execute all schedules between a defined date and the current date, then the normal schedule will be executed."
-    )
-    @PluginProperty
-    private ScheduleBackfill backfill;
```

Comment:
You should keep it as deprecated and emit a WARNING if used or you will break existing code.